sdk-rs/src/store.rs

```diff
@@ -4,33 +4,25 @@ use crate::{error::Error, Client as SdkClient};
 use base64::{engine::general_purpose, Engine as _};
 use reqwest::header::{HeaderValue, AUTHORIZATION};
 use serde::{Deserialize, Serialize};
+use serde_with::{base64::Base64, serde_as};
 
 /// The JSON payload for a `get` operation response.
+#[serde_as]
 #[derive(Serialize, Deserialize, Debug)]
 pub struct GetResultPayload {
```

Comment:
I wonder if we need to include `*Payload` on all of these?

Maybe just: `GetResult`, `QueryResultItem`, `QueryResult`, etc.

Reply:
I kind of like *Payload as an explcit indicator that the types have appropriate serialize/deserialize to be used as HTTP payloads, but don't feel super strongly.

Reply:
My approach has been to "hide the complexity of transport" from the application (so it just uses types) but I'll admit the previous art here wasn't great lol.

Let's go for a minimal diff for now (and keep as is).